src/interfaces/logfiles.js

```diff
@@ -43,10 +44,20 @@ function mountApi(app) {
       res.json(files)
     })
   })
-  app.get('/logfiles/:filename', function(req, res, next) {
+  app.get(`${SERVERROUTESPREFIX}/logfiles/:filename`, function(req, res, next) {
     const sanitizedLogfile = path
       .join(getFullLogDir(app), req.params.filename)
       .replace(/\.\./g, '')
     res.sendFile(sanitizedLogfile)
   })
+  app.get(`${SERVERROUTESPREFIX}/ziplogs`, function(req, res, next) {
```

Comment:
This would be more "resty" with path  `${SERVERROUTESPREFIX}/logfiles`

I assume res.zip sets the mime type to `application/zip`.

Reply:
Done